lib/Dialect/TTIR/Transforms/Layout.cpp

```diff
@@ -176,17 +176,21 @@ createToLayoutOp(PatternRewriter &rewriter, Location loc, Value input,
 
 static std::optional<Value>
 createToLayoutOp(PatternRewriter &rewriter, Location loc, Value input,
```

Comment:
It seems there is no point to this function anymore, you can remove it and call the latter one.